qa/cephfs/begin/1-ceph.yaml

```diff
@@ -3,4 +3,3 @@ log-rotate:
   ceph-osd: 10G
 tasks:
   - ceph:
-      create_rbd_pool: false
```

Comment:
This should be kept, there is no reason to create an RBD pool for CephFS tests.

Reply:
oh yes, I didn't add the source while pushing the signed-off commit.

Reply:
I resorted to reverting just the one commit from PR #53037.